bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -575,6 +575,15 @@ data:
         "
       fi
 
+      ovn_v4_join_subnet_opt=
+      if [[ "{{.V4JoinSubnet}}" != "" ]] ; then
```

Comment:
nit: space in `]] ;` not used elsewhere so probably remove it for consistency. Many instances of this.

Reply:
Ack